runtime/src/mm/cpp/InitializationSchemeTest.cpp

```diff
@@ -39,6 +39,7 @@ class InitSingletonTest : public testing::Test {
         // Make sure to clean everything allocated by the tests.
         for (auto& threadData : threadDatas_) {
             threadData->objectFactoryThreadQueue().ClearForTests();
+            threadData->globalsThreadQueue().ClearForTests();
```

Comment:
`globalsThreadQueue()` will publish into a global (inside `GlobalData`) when `threadData` is destroyed, so we need to clean them up before it happens. In the future I'll look into decoupling tests from accessing globals.